infrastructure/quick-deploy/localhost/all-in-one/storage.tf

```
@@ -186,4 +196,8 @@ locals {
       bucket_name = try(module.minio[0].bucket_name, "")
     } : null
   }
+  queue_module = try(coalesce(
+    length(module.activemq) > 0 ? "activemq" : null,
+    length(module.rabbitmq) > 0 ? "rabbitmq" : null,
+  ), "")
```

Either:
```suggestion
  queue_module = coalesce(one(module.activemq), one(module.rabbitmq))
```
Or:
```suggestion
  queue_module = compact(concat(module.activemq, module.rabbitmq))[0]
```

I am not sure which one is best.